### DISCUSSION TITLE
Would love a Chrome version of this!

### DISCUSSION THREAD

**Discussion by SawkeeReemo** [jlesage/docker-firefox#221] [General]
This Firefox container works really well for my needs! Thanks for making it! Any chance a Chrome version is on the horizon? I've been searching for a while to find one that's as easy to set up as yours, but haven't had any luck so far.

**Comment by jlesage** [comment]
Yes this could be possible, but I was unsure to publish one because there are a few versions that already exist.

**Comment by SawkeeReemo** [comment]
I think for me, as a non-developer, just a casual tinkerer, your container and Linuxserver.io’s container are the easiest to set up that I’ve found. Yours is actually easier, but theirs has KasmVNC as a base that somehow accepts macOS clipboard copy/paste. (If yours is able to as well, I would love to know how. Right now I have to paste into the little clipboard box on the VNC side menu.)

But due to the ease at which I can set yours up, and for my specific use case, I’ll take ease over that specific function. And I do appreciate the ease and well thought out instructions you’ve laid out here. So many of these are pretty difficult to follow for a casual tinkerer.
